Garrett-Iannuzzi/final-countdown
# Basic testing for App, Form, WordContainer, and WordBtn

## What does this PS do?
- Adds testing libraries to the `package.json`
- Tests `App` for:
1) Rendering the correct h1
- Tests `Form` for:
1) Rendering the correct btn
2) Rendering an empty input
- Tests `WordContainer` for:
1) Rendering the correct h2
2) Rendering with the correct word passed down
- Tests `WordBtn` for:
1) Rendering with the correct searchedWord 

## How to test?
- Pull down branch `basic-testing`
- Run `npm I`
- Run `npm test`
- You will see check marks for passing tests

## Issues:
- This PR will start work on #19 but not finish it
- The board is up to date 

## Screenshots
- N / A

## Notes:
- We are using the `@testing-library/jest-dom/1`

Thanks!
